journeys/deltath.md

```diff
@@ -0,0 +1,34 @@
+<h1 align="center">Hi 👋, I'm Abhay Nandan Singh</h1>

```

Comment:
Hi Abhay, The journey file is the same as your GitHub profile readme file. Can you please write more about your journey? If you want to know what exactly your journey should include please refer to [What should I share in my journey?](https://collab-community.github.io/journey-book/#/_pages/what-should-i-share).